### DISCUSSION TITLE
Will there be support for additional languages?

### DISCUSSION THREAD

**Discussion by kachalovmg** [Chevereto/Chevereto-Free#148] [Help needed]
Previously, the system had additional languages, in particular Russian.
After the update, it disappeared, or I cannot find how to enable it.
I apologize for the auto-translator.

**Comment by rodber** [comment]
Hi @kachalovmg

What you can expect from me with this project is to get maintenance and support for future PHP versions (until that's feasible).

Cheers,
Rodolfo.

**Comment by keven1024** [comment]
you can see this [fork](https://github.com/keven1024/chevereto-free-multi-language)
